runtime/onert/backend/train/TensorBuilder.cc

```diff
@@ -95,6 +95,27 @@ void TensorBuilder::registerDisposableBackwardTensorInfo(const DisposableTensorI
   _disposable_backprops.add(index);
 }
 
+void TensorBuilder::registerLayerScopeTensor(const LayerScopeTensorIndex &index,
+                                             std::shared_ptr<LayerScopeTensor> &tensor)
```

Comment:
(Optional)
It would be OK to use `const std::shared_ptr` to share(copy) ownership of the `std::shared_ptr`.

Reply:
> It would be OK to use const std::shared_ptr to share(copy) ownership of the std::shared_ptr. 

I couldn't fully understand this sentence. :cry: 
Do you mean to update this line like below? 

```suggestion
                                             const std::shared_ptr<LayerScopeTensor> &tensor)
```

Reply:
Sorry for my incorrect suggestion. I misunderstand this function as registering `LayerScopeTensor` into the data of `std::shared_ptr`.